### DISCUSSION TITLE
What do the percentiles on dv/v plot show? (development version)

### DISCUSSION THREAD

**Discussion by kjdavidson** [ROBelgium/MSNoise#340] [Q&A] [answered]
Hello, apologies if this is answered somewhere but I can't find anything in the documentation.

When I plot my final dv/v results using the development version I now see some percentiles in the output too:
![image](https://github.com/ROBelgium/MSNoise/assets/35636042/0d559945-0b99-49a4-9c41-3296091e2127)

Is there a quick explanation for what these are showing please? I've had a look around the code and can see that the dvv dataframe is grouping the results by these percentiles, but I'm not sure what they are actually showing.

Thanks for any advice on offer here!

**Comment by ThomasLecocq** [comment, marked as answer]
For each "time step" (aka previously "1-day", but today can be anything -> so for each unique timestamp on the X-axis) , you might have many station-pairs and therefore many dv/v measurements. That "set" can be "described" using stats, like mean, median, or the percentile of the distribution. Of course, most of those don't make sense if you have 1, 2 or 3 pairs, or event "few" pairs.

Note the 5% & 95% are currently reversed, because the stats are actually computed on the dt/t and not dv/v so I'll probably change that in a near future.
